backend/src/govsg/middlewares/govsg-verification.middleware.ts

```diff
@@ -99,8 +100,13 @@ export const resendPasscodeCreationMessage = async (
     })
   }
   const { recipient } = govsgMessage
+  // flamingoDb has phone numbers prepended with +countrycode but phone numbers in govsgMessage don't always have that.
+  // In addition, phone numbers in govsgMessage are sometimes observed to have whitespaces inside them. Such is not observed
+  // in flamingoDb.
```

Comment:
Is this comment clearer?

```suggestion
  // flamingoDb numbers are prepended with +countrycode and have no whitespaces.
  // recipient numbers of govsgMessage may not fulfil the above thus have to be preprocessed first.
```

Reply:
Let me take it!

Thank you Ivan!